bin/millau/runtime/src/lib.rs

```diff
@@ -311,18 +311,36 @@ parameter_types! {
 	// Note that once this is hit the pallet will essentially throttle incoming requests down to one
 	// call per block.
 	pub const MaxRequests: u32 = 50;
+	pub const WestendSessionLength: bp_westend::BlockNumber = bp_westend::SESSION_LENGTH;
+	pub const RialtoSessionLength: bp_rialto::BlockNumber = bp_rialto::SESSION_LENGTH;
+
+	// TODO [#846]: Right now this will break benchmarking if it is greater than `u8::MAX`
+	pub const RialtoValidatorCount: u64 = 255;
+	pub const WestendValidatorCount: u64 = 255;
 }
 
 pub type RialtoGrandpaInstance = ();
 impl pallet_bridge_grandpa::Config for Runtime {
 	type BridgedChain = bp_rialto::Rialto;
 	type MaxRequests = MaxRequests;
+	type BridgedSessionLength = RialtoSessionLength;
+	type BridgedValidatorCount = RialtoValidatorCount;
+
+	// NOTE: We should technically be using weights generated for the Millau runtime, but since we
+	// don't have those we'll just use the Rialto runtime ones.
+	type WeightInfo = pallet_bridge_grandpa::weights::RialtoWeight<Runtime>;
```

Comment:
I left it as a note instead of a TODO because I remembered [this issue](https://github.com/paritytech/parity-bridges-common/issues/390) related to the Millau benches. However, reading the comments again I realize that this should fall under a TODO for #391 instead.

Will update.